src/intervals/interval.jl

```diff
@@ -100,7 +118,16 @@ function isoverlapping{S, T}(a::Interval{S}, b::Interval{T})
 end
 
 function Base.show(io::IO, i::Interval)
-    print(io, i.seqname, ":", i.first, "-", i.last, "    ", i.strand, "    ", i.metadata)
+    if get(io, :compact, false)
+        print(io, i.seqname, ":", i.first, "-", i.last, "  ", i.strand, "  ", i.metadata)
+    else
+        println(io, summary(i), ':')
+        println(io, "  sequence name: ", i.seqname)
+        println(io, "  leftmost position: ", i.first)
+        println(io, "  rightmost position: ", i.last)
+        println(io, "  strand: ", i.strand)
+          print(io, "  metadata: ", i.metadata)
```

Comment:
This is intentional; the variable `io` and the string literal are beautifully aligned to the lines above 😁